### ISSUE TITLE
Quick Fix support for imports from path mappings

### ISSUE THREAD

**Issue by dartzki** [Microsoft/TypeScript#15898]
**TypeScript Version:** nightly (2.2.4-dev.20170517)

**Code**
https://github.com/dartzki/typescript-vscode-multiproject

**Expected behavior:**
The quick-fix should propose to import files from mapped paths

**Actual behavior:**
The quick-fix does not propose imports form mapped paths, only from local paths

**Comment by mhegazy** [comment]
@aozgaa can you take a look.

**Comment by Goodwine** [comment]
When you say "mapped paths" do you mean tsconfig?

I raised this problem on TypeStrong/atom-typescript#1289 and I was referred here, so I'm just trying to make sure that this is the same as that to avoid creating a dup.

**Comment by mhegazy** [comment]
This should be working now as expected. for instance `import { helloApp } from "src/app-module";` in the example of the OP should work. 
the other two should work too, but the files need to be included int he project in the first place for the compiler process them. in other words, the source files need to be in the `"include"` in the tsconfig.json

**Comment by dartzki** [comment]
@mhegazy the first one still tries to import using the relative path instead of the mapped path
actual:
`import { helloCommon } from "../../common/src/common-module";`
expected:
`import { helloCommon } from "common/common-module";`

I've updated the tsconfig.json and included the two additional src directories:
```
  "include": [
    "./src/**/*",
    "./service-files/src/**/*",
    "../common/src/**/*"
  ],
```

I'm using typescript 2.6.2

**Comment by mhegazy** [comment]
This is fixed in TS 2.7 and later. so you need to be using `typescript@next`.
